btree/btree.c

```diff
@@ -941,7 +941,8 @@ struct node_type {
 			 struct m0_be_tx *tx);
 
 	/** Deletes the record from the node at specific index */
-	void (*nt_del)  (const struct nd *node, int idx, struct m0_be_tx *tx);
+	void (*nt_del)  (const struct nd *node, int idx,
+			 struct m0_be_tx *tx);
```

Comment:
struct m0_be_tx *tx can be added to previous line

Reply:
Yes